grouper/public_key.py

```diff
@@ -233,3 +234,17 @@ def get_public_key_tag_permissions(session, tag):
     ).all()
 
     return permissions
+
+
+def generate_key_pair(ssh_type, size):
+    try:
+        subprocess.check_output(["yes | ssh-keygen -t {} -b {} -f /dev/stdout".format(ssh_type,
+        size)], shell=True)
+    # ssh-keygen will try to output the public key to /dev/stdout.pub, which doesn't exist and
```

Comment:
We could alternatively use PyCrypto. See e.g. http://stackoverflow.com/questions/2466401/how-to-generate-ssh-key-pairs-with-python